pkgs/applications/editors/emacs/elisp-packages/manual-packages/lua/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenv,
+  lua,
+  melpaBuild,
+  pkg-config,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+melpaBuild {
+  pname = "lua";
+  version = "0-unstable-2025-01-27";
+
+  src = fetchFromGitHub {
+    owner = "syohex";
+    repo = "emacs-lua";
+    rev = "501189b5fc069fcead8843b2b0ad510c08de1397";
+    hash = "sha256-psCrto12p03R9XxPtDYTMB5vcRVWj+Blq7D30nLsSbU=";
+  };
+
+  preBuild = ''
+    make LUA_VERSION=${lua.luaversion}
```

```suggestion
    make LUA_VERSION=${lua.luaversion} CC=$CC LD=$CC
```

This is necessary if you want to support darwin.